docs/specs/xref.yml

```diff
@@ -42,3 +42,92 @@ outputs:
     { "content": "<p>Link to <a href=\"https://docs.microsoft.com/en-us/dotnet/api/system.string\">String</a></p>\n"}
   build.manifest:
 ---
+# Define uid in conceptual, resolve it
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.md: |
+    ---
+    title: Title from yaml header a
+    uid: a
+    ---
+  docs/b.md: |
+    ---
+    title: Title from yaml header b
+    uid: b
+    ---
+  docs/c.md: Link to @a
+outputs:
+  docs/c.json: | 
+    {"content":"<p>Link to <a href=\"docs/a.json\">Title from yaml header a</a></p>\n"}
+  docs/a.json:
+  docs/b.json:
+  build.manifest:
+---
+# Define uid in SDP, resolve it
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "a",
+      "name": "test",
+      "fullName": "test.test",
+      "description": "a.description"
+    }
+  docs/c.md: Link to @a
+outputs:
+  docs/c.json: | 
+    {"content":"<p>Link to <a href=\"docs/a.json\">a</a></p>\n"}
```

Comment:
>a [](start = 51, length = 1)

should this be test, which is "name"?

Reply:
not setting anything in SDP xref resolving except uid for now.... will implement more logic in later PR

---
In reply to: [219087735](https://github.com/dotnet/docfx/pull/3437#discussion_r219087735) [](ancestors = 219087735)